### ISSUE TITLE
Editorial: aria-braillelabel is not in alphabetical order with other attrs

### ISSUE THREAD

**Issue by cookiecrook** [w3c/core-aam#87]
Editorial: aria-braillelabel is near aria-label in the spec attribute list, rather than in alphabetical order like the rest of the aria attributes.

**Comment by pkra** [comment]
I think this issue is about aria (not core-aam)? (I could've sworn I did that a while back because somebody pointed it out during PR review.)

**Comment by pkra** [comment]
>(I could've sworn I did that a while back because somebody pointed it out during PR review.)

Ah, I did that for brailleRoleDescription

**Comment by pkra** [comment]
Wow, that was a weird accidental close.

**Comment by joanmarie** [comment]
@cookiecrook: I agree with @pkra (unless I'm missing something). I don't see `aria-braillelabel` in https://w3c.github.io/core-aam/

Therefore, moving this to ARIA. If I am missing something, I suck and please explain. :smile:
